src/client/utils.ts

```diff
@@ -1,6 +1,32 @@
 import * as vscode from 'vscode';
+import { exec, ExecOptions, ExecException } from 'child_process';
```

Comment:
```suggestion
import { exec } from 'child_process';
```
Remove unused imports.